impeller/display_list/aiks_dl_vertices_unittests.cc

```diff
@@ -379,7 +376,15 @@ TEST_P(AiksTest, DrawVerticesWithInvalidIndices) {
   builder.DrawRect(SkRect::MakeLTRB(0, 0, 400, 400), paint);
   builder.DrawVertices(vertices, flutter::DlBlendMode::kSrc, paint);
 
-  ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
+  if (GetBackend() == PlaygroundBackend::kOpenGLES) {
```

Comment:
I would do this on all backends, since we don't care about the output ever - just that it doen't implode.

Reply:
done